### DISCUSSION TITLE
Jupyter Lab or Notebook CELL output behavior is different with Python & R Languages via Anaconda 3

### DISCUSSION THREAD

**Discussion by mbennette68** [jupyter/notebook#7203] [Q&A]
Hello,

I am using Jupyter with Anaconda3. 

When using Jupyter Lab/Notebook with Python, a vector displays, Left to Right. 
When using Jupyter Lab/Notebook with R Language, the vector displays, Top to Bottom.
This behavior is shown on the image that follows. Left side of image runs Python Notebook, while right side runs R language.

![image](https://github.com/jupyter/notebook/assets/16678277/5e1a9d55-f896-40bd-b5fd-b01e2a239aaa)
May I ask if there is any setting that I can change so that R's output in Jupyter notebook behaves like python output without 
having to activate the notebook's Render Side by Side setting which is not what I'm looking to do? 

Basically, I am wondering why I can't see the same behavior in the Notebook Cell that is present in the R Console.
![image](https://github.com/jupyter/notebook/assets/16678277/9dd0e414-8212-4ab9-83b6-4c1ee7b5cd2d)

I've been looking on the Web for anything related to this and have looked at the settings but have not had any luck finding a way to make R render left to right in Jupyter.

Thank you,
mb
